test.js

```diff
@@ -1,36 +1,59 @@
 'use strict';
-var assert = require('assert');
+var assert = require('chai').assert;
```

Comment:
Should I disable the specific version test for the scoped package, and log a message to the command line if a user tries to do so for now so that we can get the rest of this patch deployed? We could then enable specific version functionality for scoped packages in the future when the downstream issue with npm is resolved?